container_linux_test.go

```diff
@@ -380,3 +380,137 @@ func TestContainerAttach(t *testing.T) {
 		t.Errorf("expected output %q but received %q", expected, output)
 	}
 }
+
+func TestContainerAttachProcess(t *testing.T) {
```

Comment:
It should, I have to implement DirectIO on windows first for these tests, I have it on my todo list for the Attach tests